lib/std/Uri.zig

```diff
@@ -251,7 +243,7 @@ pub fn format(
 
         if (uri.query) |q| {
```

Comment:
If we do this change, i'd rename `query` to `raw_query` to inform the user that it is actually not containing any escaped data